mkosi/sandbox.py

```diff
@@ -871,11 +871,11 @@ def execute(self, oldroot: str, newroot: str) -> None:
 """
 
 
-def main() -> None:
+def main(argv: list[str] = []) -> None:
```

Comment:
Shouldn't it be fine to move `sys.argv` up here?
```suggestion
def main(argv: list[str] = sys.argv[1:]) -> None:
```